Source/WebKit/UIProcess/ProcessAssertion.h

```diff
@@ -100,6 +102,7 @@ class ProcessAssertion : public ThreadSafeRefCounted<ProcessAssertion>, public C
     RetainPtr<RBSAssertion> m_rbsAssertion;
     RetainPtr<WKRBSAssertionDelegate> m_delegate;
     bool m_wasInvalidated { false };
+    WebCore::Timer m_acquireTimer;
```

Comment:
Also, WebCore::Timer in the UIProcess is a big NO-NO because apps may use both WK1 (with WebThread) and WK2.

Reply:
RunLoop::Timer is the timer that's safe to use in the UI Process.